src/map/clif.hpp

```diff
@@ -905,8 +905,8 @@ void clif_parse_LoadEndAck(int fd,map_session_data *sd);
 void clif_hotkeys_send(map_session_data *sd, int tab);
 
 // trade
-void clif_traderequest(map_session_data* sd, const char* name);
-void clif_tradestart(map_session_data* sd, uint8 type);
+void clif_traderequest(map_session_data& sd, const char* name);
+void clif_traderesponse(map_session_data& sd, uint8 type);
```

Comment:
```suggestion
void clif_traderesponse( map_session_data& sd, e_ack_trade_response result );
```

Reply:
compilation failed :/

Reply:
i think is because of trade.hpp not included in cliff.hpp but in cliff.cpp, need to checkit